app/components/DocsPopover.tsx

```diff
@@ -46,11 +46,10 @@ export const DocsPopover = ({ heading, icon, summary, links }: DocsPopoverProps)
   const title = `Learn about ${heading}`
   return (
     <Popover>
-      <PopoverButton
-        className={cn(buttonStyle({ size: 'sm', variant: 'ghost' }), 'w-8')}
-        title={title}
-      >
-        <Info16Icon aria-hidden className="shrink-0" />
+      <PopoverButton title={title}>
+        <div className={cn(buttonStyle({ size: 'sm', variant: 'ghost' }), 'w-8')}>
+          <Info16Icon aria-hidden className="shrink-0" />
+        </div>
```

Comment:
This one doesn't work, not sure why. When I move `buttonStyle` to the `PopoverButton`, I can get it to move when I check `:active` in the dev tools, but that doesn't happen when I actually click it.

Reply:
This works for me, can you still reproduce?

Reply:
Turns out it's only broken in Firefox. I think we can live with that.